DNN Platform/Modules/HTML/EditHtml.ascx.cs

```diff
@@ -270,6 +270,7 @@ protected void OnSaveClick(object sender, EventArgs e)
                         }
 
                         this._htmlTextController.UpdateHtmlText(htmlContent, this._htmlTextController.GetMaximumVersionHistory(this.PortalId));
+                        Entities.EventManager.Instance.OnModuleGenericEvent(new Entities.Modules.Actions.ModuleGenericEventArgs { Module = this.ModuleConfiguration, EventName = "Update", Payload = htmlContent });

```

Comment:
I would think we'd want to set an example of having a unique event name, e.g. `DNN_HTML_Update`

Reply:
Given that the module info is sent along we already know the context. So I can't see the point of asking module developers to adhere to a central naming scheme. Every module developer can create their own little world of conventions, IMHO.